source/CommandLine/Plumbing/IOctopusFileSystem.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace Octopus.CommandLine.Plumbing
+{
+    public interface IOctopusFileSystem
```

Comment:
Really trimmed this down to the bare minimum.
This might be a candidate for pulling into it's own library.

Reply:
TIL about `System.IO.Abstractions`... Looks cool. I think we've got some custom code in there though, so it wouldn't be drop in.